options.go

```diff
@@ -371,3 +372,54 @@ func ExpandPath(path string) string {
 	}
 	return abspath
 }
+
+// Env inits environment names for flags.
+// For example:
+//   --some.value -> PREFIX_SOME_VALUE
+func Env(prefix string) Option {
+	processFlag := func(flag *Flag) {
+		switch env := flag.Env; {
+		case flag.Name == "help":
+			return
+		case env == "-":
+			flag.Env = ""
+			return
+		case env != "":
+			return
+		}
+		replacer := strings.NewReplacer("-", "_", ".", "_")
+		name := replacer.Replace(flag.Name)
+		// Split by upper chars "SomeOne" -> ["Some", "One"]
```

Comment:
Please replace this with the existing function that Kong includes for splitting camel case ([camelCase](https://github.com/alecthomas/kong/blob/master/camelcase.go#L47)?